h2o-docs/src/product/data-science/gam.rst

```diff
@@ -136,17 +140,17 @@ Defining a GAM Model
 
 -  `export_checkpoints_dir <algo-params/export_checkpoints_dir.html>`__: Specify a directory to which generated models will automatically be exported.
 
--  **k**: An array that specifies the number of knots for each predictor.
+-  **num_knots**: An array that specifies the number of knots for each predictor.
             
--  **knots_key**: A string array storing frame keys that contain knot locations. Specify one value for each GAM column specified in ``gam_x``.
+-  **knot_ids**: A string array storing frame keys/IDs that contain knot locations. Specify one value for each GAM column specified in ``gam_columns``.
 
--  **gam_x**: A predictor column names array.
+-  **gam_columns**: Required. An array specifying the predictor column names for GAM.
 
```

Comment:
@angela0xdata 
For this one: **gam_columns**: Required. An array specifying the predictor column names for GAM.

what I want it to convene is that gam_columns contain predictor names that the user want to choose for us for build a smoother for.  Please translate that to whatever wording you deem fit.

Reply:
@angela0xdata:  Perfect!  Thanks!